apps/apps.humanatlas.io/src/app/pages/us1/cell-population-predictor/cell-population-predictor.component.scss

```diff
@@ -19,6 +19,7 @@
   flex-wrap: wrap;
   gap: 3rem;
   margin-top: 3.5rem;
+  height: 15.8125rem;
```

Comment:
There height changes incorrectly affects the responsiveness of the cards and page. Check the app at around 680px width and you'll see the problem